src/net.h

```diff
@@ -213,36 +213,43 @@ class CConnman
     void ForEachNode(Callable&& func)
     {
         LOCK(cs_vNodes);
-        for (auto&& node : vNodes)
-            func(node);
+        for (auto&& node : vNodes) {
```

Comment:
It's conducive to large type changes, though. With this one in particular, when CNode becomes a shared_ptr\<CNode\>, this code requires no changes.

For exactly that reason, I actually have local changes which switch _all_ loop operations on vNodes to ```for (auto&& node : vNodes)``` :\